### PR TITLE
Documented the "values" option of the Expression constraint

### LINKED ISSUE #9098: [Validator] Add option to pass custom values to Expression validator
See https://github.com/symfony/symfony/pull/25504.

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #9098.

@ostrolucky I have problems defining the XML config of this example. Could you please help me? Thanks!

### CODE REVIEW

**Comment by ostrolucky** [comment]
I'm not knowledgable in XML, but this reminds me same issue we had with Workflow metadata. @stof responded with this https://github.com/symfony/symfony/issues/23257#issuecomment-312025290, seems it could be applied here too

**Comment by javiereguiluz** [comment]
@ostrolucky thanks for your help! I did some changes, but we'll need a review here from some XML expert. Thanks!

File: reference/constraints/Expression.rst [hunk 1]
```diff
@@ -253,3 +254,92 @@ message
 The default message supplied when the expression evaluates to false.
 
 .. include:: /reference/constraints/_payload-option.rst.inc
+
+values
+~~~~~~
+
+**type**: ``array`` **default**: an empty array
```
Comment (wouterj) [inline reference/constraints/Expression.rst]: Don't we use `[]` in that case? (or just nothing, as empty is like no default)

File: reference/constraints/Expression.rst [hunk 2]
```diff
@@ -253,3 +254,92 @@ message
...
+    .. code-block:: php-annotations
+
+        // src/Model/Analysis.php
+        namespace App\Model;
+
+        use Symfony\Component\Validator\Constraints as Assert;
+
+        class Analysis
+        {
+            // ...
```
Comment (wouterj) [inline reference/constraints/Expression.rst]: imo, this and the empty line below can be removed. That saves some lines

**Review by wouterj — APPROVED** [review]

**Comment by wouterj** [comment]
I can confirm the XML is correct

status: reviewed

**Review by ostrolucky — APPROVED** [review]

**Comment by ostrolucky** [comment]
Tried this xml, works

**Pull request closed without merging**